web_m2x_options/static/src/js/form.js

```diff
@@ -100,8 +100,19 @@ openerp.web_m2x_options = function (instance) {
 
             var create_rights;
             if (!(self.options && (self.options.no_create || self.options.no_create_edit))) {
-                create_rights = new instance.web.Model(this.field.relation).call(
-                    "check_access_rights", ["create", false]);
+                // check quick create options
+                var target_model = this.field.relation
+                create_rights = new instance.web.Model('ir.model').
```

Comment:
It's an extra RPC call of course, but I don't think it will be noticeable. This is only in the form view and I think the number of m2x fields per view isn't that big that this should be a concern here. 
I don't have an idea how to cache here. Any suggestions?